### DISCUSSION TITLE
RIME输入法安装脚本，这个外观是怎么实现的？

### DISCUSSION THREAD

**Discussion by kwindva** [rime/home#1723] [Q&A]
https://github.com/Mark24Code/rime-auto-deploy/tree/main?tab=readme-ov-file
我看这个外观好好看，怎么能把预编辑区配置成那样？
我的预编辑区就只是一个下划线而已。。还跟已经打出来的文字对不齐。。
<img width="681" alt="result" src="https://github.com/user-attachments/assets/18706453-6a87-4bd7-b663-3e649683ef16">

**Comment by Traveler07** [comment]
翻一下配置文件，里面有写

**Comment by lotem** [comment]
樓主說的預編輯區那部分應該屬於應用程序的界面。
可能編輯器的選定區域就是這個風格。

**Comment by heixxxa** [comment]
Windows上好像没有办法设置预编辑区域
